cms/static/sass/views/_group-configuration.scss

```diff
@@ -19,206 +19,200 @@
       text-align: center;
     }
 
-    .no-group-configurations-content {
-      @extend %ui-well;
-      padding: ($baseline*2);
-      background-color: $gray-l4;
-      text-align: center;
-      color: $gray;
+    .no-content {
+      @extend %no-content;
     }
 
-    .new-button {
-      @extend %t-action3;
-      margin-left: $baseline;
+    .wrapper-groups {
+      margin-bottom: ($baseline*1.5);
 
-      .icon {
-        margin-right: ($baseline/2);
+      .title {
+        @extend %t-title4;
+        @extend %t-strong;
+        margin-bottom: ($baseline/2);
+      }
+
+      .copy {
+        @extend %t-copy-sub1;
       }
     }
 
-    .group-configuration {
+    .wrapper-collection {
```

Comment:
Nice abstraction! Definitely not needed to merge this work, but let's make a note/story to move this styling to a more general place that can be called upon by other views (either in consistent class names or by Sass extends).